src/opf/acp.jl

```diff
@@ -1,293 +1,167 @@
 struct ACOPF <: AbstractFormulation end
 
 """
-    build_acopf(data, optimizer)
+    build_opf(ACOPF, data, optimizer)
 
-Build an AC-OPF model.
-
-This implementation is based on the AC-OPF formulation of Rosetta-OPF
-    https://github.com/lanl-ansi/rosetta-opf/blob/38a951326df3156d79dcdc49c8010aa29905b05d/jump.jl
+Build an ACOPF model.
 """
-function build_opf(::Type{ACOPF}, data::Dict{String,Any}, optimizer;
+function build_opf(::Type{ACOPF}, network::Dict{String,Any}, optimizer;
     T=Float64,    
 )
-    # Cleanup and pre-process data
-    PM.standardize_cost_terms!(data, order=2)
-    PM.calc_thermal_limits!(data)
-    ref = PM.build_ref(data)[:it][:pm][:nw][0]
-
-    # Check that slack bus is unique
-    length(ref[:ref_buses]) == 1 || error("Data has $(length(ref[:ref_buses])) slack buses (expected 1).")
-    i0 = first(keys(ref[:ref_buses]))
+    # TODO: remove when all formulations are done
+    data = OPFData(network)
     
     # Grab some data
-    N = length(ref[:bus])
-    G = length(ref[:gen])
-    E = length(data["branch"])
-    L = length(ref[:load])
-    bus_loads = [
-        [ref[:load][l] for l in ref[:bus_loads][i]]
-        for i in 1:N
-    ]
-    bus_shunts = [
-        [ref[:shunt][s] for s in ref[:bus_shunts][i]]
-        for i in 1:N
-    ]
+    N, E, G = data.N, data.E, data.G
+    vmin, vmax = data.vmin, data.vmax
+    i0 = data.ref_bus
+    gs, bs = data.gs, data.bs
+    pd, qd = data.pd, data.qd
+    bus_arcs_fr, bus_arcs_to = data.bus_arcs_fr, data.bus_arcs_to
+    bus_gens = data.bus_gens
+    pgmin, pgmax = data.pgmin, data.pgmax
+    qgmin, qgmax = data.qgmin, data.qgmax
+    c0, c1, c2 = data.c0, data.c1, data.c2
+    gen_status = data.gen_status
+    bus_fr, bus_to = data.bus_fr, data.bus_to
+    gff, gft, gtf, gtt = data.gff, data.gft, data.gtf, data.gtt
+    bff, bft, btf, btt = data.bff, data.bft, data.btf, data.btt
+    dvamin, dvamax, smax = data.dvamin, data.dvamax, data.smax
+    branch_status = data.branch_status
 
     model = JuMP.GenericModel{T}(optimizer)
-    model.ext[:opf_model] = ACOPF  # for internal checks
+    model.ext[:opf_model] = ACOPF
 
     #
     #   I. Variables
     #
+    
     # nodal voltage
-    JuMP.@variable(model, va[1:N])
-    JuMP.@variable(model, ref[:bus][i]["vmin"] <= vm[i in 1:N] <= ref[:bus][i]["vmax"], start=1.0)
+    @variable(model, vm[i in 1:N], start=1.0)
```

Comment:
`i` is not needed since we no longer set the lower/upper bounds here
```suggestion
    @variable(model, vm[1:N], start=1.0)
```